m4_sessions/1804inning/treating_depression_with_mobile_app.md

```diff
@@ -5,12 +5,39 @@ Andrew Jeffery
 Kailin Cannon
 Matt Renn
 
-Name of Session: 
+Name of Session: Treating Depression with Mobilie Apps
 
 Session Description (Why should students come to this session):
+This session will explore some the mobile treatment options as well as technological sources.  Students will leave with tools to help themselves an others.
 
 Session Outcomes (What will students learn/leave with?):
 
+* What can I do to help myself
+* I am now empowered to help others
+
 Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+Pre Gear-Up:
+Send Resources
+
+* Intro:
+- Name and Mod (5 mins)
+- small group intros (5 mins)
+- Why did you choose this session?
+
+* Scope of the coversation (5 mins)
+- types of depression and what we are focusing on - Moderate, not severe.
```

Comment:
Good--narrowing the scope is important.

Reply:
Maybe even a comment on severe types should seek different help than a mobile app.